upgrade/util/util.go

```diff
@@ -1198,6 +1224,26 @@ func updateOrphans(namespace string, lhClient *lhclientset.Clientset, orphans ma
 	return nil
 }
 
+func updateSystemBackups(namespace string, lhClient *lhclientset.Clientset, systemBackups map[string]*longhorn.SystemBackup) error {
```

Comment:
`orphans` is misleading. just `systemBackups`

Reply:
Modified. Thanks.